multimodal/src/autogluon/multimodal/data/utils.py

```diff
@@ -212,3 +214,29 @@ def apply_data_processor(modality_features: dict, data_processors: dict, idx: in
                 )
 
     return sample_features
+
+
+def register_encoding_decoding_error_handlers() -> None:
+    """Register the encoding and decoding error handlers for `utf-8` and `cp1252`."""
+
+    def replace_encoding_with_utf8(error: UnicodeError) -> Tuple[bytes, int]:
+        return error.object[error.start : error.end].encode("utf-8"), error.end
+
+    def replace_decoding_with_cp1252(error: UnicodeError) -> Tuple[str, int]:
+        return error.object[error.start : error.end].decode("cp1252"), error.end
+
+    codecs.register_error("replace_encoding_with_utf8", replace_encoding_with_utf8)
+    codecs.register_error("replace_decoding_with_cp1252", replace_decoding_with_cp1252)
+
+
+def normalize_text(text: str) -> str:
+    """Resolve the encoding problems and normalize the abnormal characters."""
```

Comment:
Maybe add a short example in the docstring here to show what text normalization does and how can it help downstream usage. But if this is a common term in NLP which requires no further explanation, it is also fine.

Reply:
Good idea! Do you think it would be fine if I add a [link](https://github.com/awslabs/autogluon/tree/master/examples/automm/kaggle_feedback_prize#15-a-few-examples-of-normalized-texts) to the example I added before

Reply:
Sure, that would be good